### DISCUSSION TITLE
Thank you for adding the ability to change the background color

### DISCUSSION THREAD

**Discussion by yogesch** [pgmodeler/pgmodeler#1548] [Ideas]
In the past my biggest problem with pgmodeler was the lack of a dark mode. When every other app was in dark mode, and this was the only one with a bright background it really hurt my eyes. So slowly I moved away from this tool that was good in every other way. 

Now it is possible for me to change the background colors. There is no official dark mode with a separate stylesheet, but I can change some of the text field colors and achieve the same effect. I just set the background color to dark grey, the grid lines to medium gray, and the table body color to medium gray, and it looks perfect! The blue bars (I made them a little darker shade of blue) on the table headers are a nice colorful touch. Almost matching with the OS dark mode colors. 

I spent over half a day trying other alternatives software but none was good as this.  So I am very happy to be able to use pgmodeler again, just in time for a new personal project. 

Couple of small further suggestions (nice to have. not need to have) in this area - it will be nice to be able to transfer the colors settings from one file to another. An actual dark mode will also be good in the future. A good UI designer can make a nice dark mode default stylesheet for the project (I am happy to help to find a designer to contribute to open source). But for now, the ability to change all the colors should be baked into the next stable release.

**Comment by rkhaotix** [comment]
> Now it is possible for me to change the background colors. There is no official dark mode with a separate stylesheet, but I can change some of the text field colors and achieve the same effect. I just set the background color to dark grey, the grid lines to medium gray, and the table body color to medium gray, and it looks perfect! The blue bars (I made them a little darker shade of blue) on the table headers are a nice colorful touch. Almost matching with the OS dark mode colors.

Yeah, the dark mode is something requested long ago... I added the ability to change canvas color in 0.9.4-alpha only as a temporary workaround. The user still has to configure colors manually. But I will add a true dark mode on pgModeler 1.0, where the user will just need to select between dark and default mode so pgModeler will do the rest to apply colors and styles. I don't want to apply my efforts to implement dark mode in 0.9.4 because in 1.0 I'll do a complete UI remodeling so a lot of things will be changed, so implementing the dark mode now just to change it in a near future doesn't make sense. :)

> I spent over half a day trying other alternatives software but none was good as this. So I am very happy to be able to use pgmodeler again, just in time for a new personal project.

I'm glad to read this. It's good to see that my effort is calling users back to pgModeler! :D

> Couple of small further suggestions (nice to have. not need to have) in this area - it will be nice to be able to transfer the colors settings from one file to another. An actual dark mode will also be good in the future. A good UI designer can make a nice dark mode default stylesheet for the project (I am happy to help to find a designer to contribute to open source). But for now, the ability to change all the colors should be baked into the next stable release.

As I said, in pgModeler 1.0 all these issues will be addressed. Thanks for offering help! :)
